src/utils.jl

```diff
@@ -26,16 +26,16 @@ function to_function(@nospecialize(x))
 end
 
 """
-    method = whichtt(tt)
+    method = whichtt(tt, mt = nothing)
 
 Like `which` except it operates on the complete tuple-type `tt`,
 and doesn't throw when there is no matching method.
 """
-function whichtt(@nospecialize(tt))
+function whichtt(@nospecialize(tt), mt::Union{Nothing, MethodTable} = nothing)
     # TODO: provide explicit control over world age? In case we ever need to call "old" methods.
     # branch on https://github.com/JuliaLang/julia/pull/44515
     # for now, actual code execution doesn't ever need to consider overlayed method table
```

Comment:
Seems like this line of the comment could be deleted?

Reply:
Yeah, JuliaInterpreter hasn't implemented the infrastructure to run the overlay method yet, but it's just not implemented yet, and it's easy to implement. I'm planning to implement it for the use case in JET.

Reply:
xref: https://github.com/JuliaDebug/JuliaInterpreter.jl/pull/682 (You don't need this at all to complete this PR. Just for reference).

Reply:
I updated the comment to link to that PR, which can then be deleted there.